<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RegisterDialog</class>
    <widget class="QDialog" name="RegisterDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>450</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Register New User</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="text">
                        <string>&lt;h2&gt;Register New Account&lt;/h2&gt;</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>User Information</string>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="usernameLabel">
                                <property name="text">
                                    <string>Username:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="usernameEdit">
                                <property name="placeholderText">
                                    <string>Login name</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="passwordLabel">
                                <property name="text">
                                    <string>Password:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="passwordEdit">
                                <property name="echoMode">
                                    <enum>QLineEdit::Password</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="confirmLabel">
                                <property name="text">
                                    <string>Confirm Password:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QLineEdit" name="confirmEdit">
                                <property name="echoMode">
                                    <enum>QLineEdit::Password</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QLabel" name="fullnameLabel">
                                <property name="text">
                                    <string>Full Name:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QLineEdit" name="fullnameEdit">
                                <property name="placeholderText">
                                    <string>Your full name</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="0">
                            <widget class="QLabel" name="departmentLabel">
                                <property name="text">
                                    <string>Department:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="4" column="1">
                            <widget class="QComboBox" name="departmentCombo">
                                <item>
                                    <property name="text">
                                        <string>IT Department</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>HR Department</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Sales Department</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Marketing Department</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Finance Department</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>Management</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                        <item row="5" column="0">
                            <widget class="QLabel" name="positionLabel">
                                <property name="text">
                                    <string>Position:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="5" column="1">
                            <widget class="QLineEdit" name="positionEdit">
                                <property name="placeholderText">
                                    <string>Your position/title</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="registerButton">
                            <property name="text">
                                <string>Register</string>
                            </property>
                            <property name="default">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="cancelButton">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>